cmake/FindSUNDIALS.cmake

```
@@ -68,9 +68,10 @@ if (SUNDIALS_DEBUG)
     " SUNDIALS_nvecparallel_LIBRARY = ${SUNDIALS_nvecparallel_LIBRARY}")
 endif()
 
-if (SUNDIALS_nvecparallel_LIBRARY)
-  list(APPEND SUNDIALS_LIBRARIES "${SUNDIALS_nvecparallel_LIBRARY}")
+if (${SUNDIALS_nvecparallel_LIBRARY} STREQUAL "SUNDIALS_nvecparallel_LIBRARY-NOTFOUND")
```

I think the following should be sufficient:

```suggestion
if (NOT SUNDIALS_nvecparallel_LIBRARY)
```

Variables that end in `-NOTFOUND` are considered false.